packages/core/src/types/IImageCalibration.ts

```diff
@@ -5,11 +5,23 @@ import CalibrationTypes from '../enums/CalibrationTypes';
  * of image calibration.
  */
 export interface IImageCalibration {
-  /** The pixel spacing for the image, in mm between pixel centers */
+  /**
+   * The pixel spacing for the image, in mm between pixel centers
+   * These are not required, and are deprecated in favour of getting the original
+   * image spacing and then applying the transforms.  The values here should
+   * be identical to original spacing.
+   */
   rowPixelSpacing?: number;
   columnPixelSpacing?: number;
   /** The scaling of this image - new spacing = original pixelSpacing/scale */
   scale?: number;
+  /**
+   * The aspect ratio of the screen.
+   * Defaults to 1 if not specified.
+   * Not well handled currently as changing the aspect ratio does not result in
+   * updating measurements in any meaningful way.
+   */
+  aspect?: number;
```

Comment:
I suggest either aspectRatio or pixelAspectRatio as I searched inside dicom standard

Reply:
I added documentation to the aspectRatio indicating it is neither of the DICOM standard ones - it is the measurement aspect ratio as compared to display screen ratio.